File: src/BetterLyrics.Core/Models/Lyrics/LyricsSearchResult.cs
namespace BetterLyrics.Core.Models.Lyrics;

public record LyricsSearchResult(
    string? Title,
    string? Artist,
    string? Album,
    double? Duration,
    string? Raw,
    string? Translation = null,
    string? Transliteration = null,
    string? Reference = null);

File: src/BetterLyrics.KMP/shared/src/commonMain/kotlin/com/jayfunc/betterlyrics/sdk/models/lyrics/LyricsSearchResult.kt
package com.jayfunc.betterlyrics.sdk.models.lyrics

data class LyricsSearchResult(
    val title: String?,
    val artist: String?,
    val album: String?,
    val duration: Double?,
    val raw: String?,
    val translation: String? = null,
    val transliteration: String? = null,
    val reference: String? = null
)